temba/msgs/models.py

```diff
@@ -1167,12 +1164,13 @@ def create_incoming(cls, channel, urn, text, user=None, date=None, org=None, con
         if contact_urn:
             contact_urn.update_affinity(channel)
 
+        topup_id = None
+
```

Comment:
on the topup_id ? that's actually original code, i just moved it up when there was an if statement, but now thats gone so I'll just revert it to the old place